swarm/swarm.go

```diff
@@ -213,9 +226,28 @@ func (self *Swarm) Start(srv *p2p.Server) error {
 		}
 	}
 
+	go self.metricsLoop()
+
+	utils.Increment("stack.start")
 	return nil
 }
 
+// metricsLoop periodically sends metrics about storage
+func (self *Swarm) metricsLoop() {
```

Comment:
You could just use `Meter` go-metrics type for this purpose, without having to manually spin up goroutines, timers and tickers.